imageio/imageio-tiff/src/test/java/com/twelvemonkeys/imageio/plugins/tiff/TIFFImageReaderTest.java

```diff
@@ -939,6 +939,16 @@ public void testStreamMetadataMM() throws IOException {
         }
     }
 
+    @Test
+    public void testCCITTTypeDetectionFillOrderRightToLeft() throws IOException {
+        ImageReader reader = createReader();
+
+        try (ImageInputStream stream = ImageIO.createImageInputStream(getClassLoaderResource("/tiff/ccitt/G32DS.tif"))) {
+            reader.setInput(stream);
+            reader.read(0);
+        }
+    }
+
```

Comment:
👍🏻 
Could probably also just add it to the test file set at the top, but this is fine for now. 😀